bookwyrm/views/search.py

```diff
@@ -123,3 +125,35 @@ def list_search(query, viewer, *_):
         )
         .order_by("-similarity")
     ), None
+
+
+def isbn_check(query):
+    """isbn10 or isbn13 check, if so remove separators"""
+    if query:
```

Comment:
This will be just a bit easier to read if you return when there is no query right at the top, so the body of the function doesn't have to be embedded in the if statement.

Something like:
``` python
if not query:
    None

su_num = re....
```